### ISSUE TITLE
Jenkins operator on Google Kubernetes engine?

### ISSUE THREAD

**Issue by atrillanes1972** [jenkinsci/kubernetes-operator#16]
Hello everybody,
Just curious if anybody had succeeded implementing this on google kubernetes engine? My tests there fails on i/o timeout, even with the changes on 'initialDelaySeconds' and 'failureThreshold' to 120 and 12 respectively.
By the way, I have also tried testing this on Docker for Windows, it fails also on creating the jenkins instance, repeating the 'Creating...' then 'running'..then 'terminating'...sequences.
Any ideas?

**Comment by tomaszsek** [comment]
@atrillanes1972 Could you provide logs form operator, Jenkins and `kubectl get events`?

**Comment by atrillanes1972** [comment]
@tomaszsek ,thanks for the response.
Here's my initial run of the first command:
$ kubectl apply -f deploy/crds/jenkinsio_v1alpha1_jenkins_crd.yaml
error: unable to recognize "deploy/crds/jenkinsio_v1alpha1_jenkins_crd.yaml": Get https://35.227.30.120/api?timeout=32s: dial tcp 35.227.30.120:443: i/o timeout
$
Here's the output of the '... get events':
$ kubectl get events --namespace all-namespaces
Unable to connect to the server: dial tcp 35.227.30.120:443: i/o timeout
$

**Comment by tomaszsek** [comment]
@atrillanes1972 It looks like your kubectl config is broken.

**Comment by atrillanes1972** [comment]
@tomaszsek , here are my config files:
--> the crd file:
apiVersion: apiextensions.k8s.io/v1beta1
kind: CustomResourceDefinition
metadata:
  name: jenkins.jenkins.io
spec:
  group: jenkins.io
  names:
    kind: Jenkins
    listKind: JenkinsList
    plural: jenkins
    singular: jenkins
  scope: Namespaced
  version: v1alpha1

--> the cr file:
apiVersion: jenkins.io/v1alpha1
kind: Jenkins
metadata:
  name: example
spec:
  master:
   image: jenkins/jenkins:lts
   readinessProbe:
     httpGet:
       path: /login
       port: 8080
       scheme: HTTP
     failureThreshold: 12
     initialDelaySeconds: 120
     periodSeconds: 10
     successThreshold: 1
     timeoutSeconds: 5
   livenessProbe:
     httpGet:
       path: /login
       port: 8080
       scheme: HTTP
     initialDelaySeconds: 120
     failureThreshold: 12
     periodSeconds: 10
     successThreshold: 1
     timeoutSeconds: 5
  seedJobs:
  - id: jenkins-operator
    targets: "cicd/jobs/*.jenkins"
    description: "Jenkins Operator repository"
    repositoryBranch: master
    repositoryUrl: https://github.com/jenkinsci/kubernetes-operator.git

**Comment by atrillanes1972** [comment]
Sorry there's a '-' dash in front of id: jenkins-operator, that gets interpreted by github as a bullet point, anyway I have removed that dash (thinking it was the one corrupting the yaml file), it still times out.

**Comment by tomaszsek** [comment]
`https://35.227.30.120/api?timeout=32s: dial tcp 35.227.30.120:443: i/o timeout` means you can't connect to Kubernetes API at all. It is not relevant to Jenkins operator. I think event simple `kubectl cluster-info` won't work. First please make sure you have valid configuration `kubectl config view`.

**Comment by atrillanes1972** [comment]
Hi @tomaszsek , turns out I have missed a step or two preparing my cluster, thanks a lot for the help. My fix is to wipe all my VMs and configs at GKE, then start from scratch the link: https://cloud.google.com/solutions/jenkins-on-kubernetes-engine-tutorial , just stopping short of installing Jenkins itself. Then when the cluster is prepared (actually I overstepped, by installing Helm, which I bet is not needed by the operator) , I followed this link: https://medium.com/virtuslab/kubernetes-native-jenkins-operator-cbdfbbecf744 , now I have a working Jenkins dashboard! Thanks!
